src/calendar/tests/unit/Calendar.ts

```diff
@@ -20,11 +20,9 @@ const compareId = compareProperty((value: any) => {
 	return typeof value === 'string';
 });
 
-let dateIndex = -1;
 const expectedDateCell = function(widget: any, date: number, active: boolean) {
-	dateIndex++;
 	return w(CalendarCell, {
-		key: `date-${dateIndex}`,
+		key: `date-${date}${active ? '' : '-dimmed'}`,
```

Comment:
Why was this change necessary?

Reply:
when I moved rendering the actual date cell into its own function (`renderDateCell`), I didn't have access to the index any more, and it seemed overkill to pass it in just for the key, when I could also generate a unique key based on the date and currentMonth.